lib/package.rb

```diff
@@ -6,8 +6,9 @@ class Package
            :git_branch, :git_hashtag
 
   boolean_property = %i[conflicts_ok git_fetchtags gnome is_fake is_musl
-                        is_static no_compile_needed no_env_options no_fhs 
-                        no_patchelf no_zstd patchelf]
+                        is_static no_compile_needed no_env_options no_fhs
+                        no_patchelf no_zstd patchelf git_clone_deep
+                        git_no_submodules]
```

Comment:
Would `no_git_submodules` be better to be consistent with the others that start with `no_`?

Reply:
It would also be nice if we had a command to print all these boolean properties.  At the very least, it should be documented somewhere other than here.

Reply:
> Would `no_git_submodules` be better to be consistent with the others that start with `no_`?

yes, i'll change that in my next commit.